mongomock/collection.py

```diff
@@ -1847,6 +1848,16 @@ def _resolve_sort_key(key, doc):
     return 1, value
 
 
+def _fix_sort_key(key_getter):
+    def fixed_getter(doc):
+        key = key_getter(doc)
+        # Convert dictionaries to make sorted() work in Python 3.
+        if isinstance(key, dict):
+            return [(k, v) for (k, v) in sorted(key.items())]
```

Comment:
In Python 2 `{'a': 1, 'b': 1} > {'a': 2}, so this will change the previous order. Is that intended, was the order broken before?